tests/karma/unit/controllers/contacts-content.js

```diff
@@ -2,11 +2,17 @@ describe('ContactsContentCtrl', () => {
   'use strict';
 
   let assert = chai.assert,
-      controller,
-      stateParams,
-      scope,
-      contactViewModelGenerator,
-      tasksForContact;
```

Comment:
I've noticed changes like this before - do you have some auto-formatting on in your editor? Generally we align the variable names so they line up under the first one (as above) rather than two spaces in (as below). It's not a big deal but if you're formatting one way and others are formatting another way then we will end up with lots of unnecessary git history entries.

Reply:
Yea, sorry. My IDE does auto-format but I usually observe and revert the changes. I missed it this time. Sorry.